StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentMethodRowButtonSnapshotTests.swift

```diff
@@ -49,6 +48,14 @@ class PaymentMethodRowButtonSnapshotTests: STPSnapshotTestCase {
         rowButton.state = .selected
         verify(rowButton)
     }
+    
+    func testPaymentMethodRowButton_ignoresCheckbox_notEmbedded() {
+        var appearance = PaymentSheet.Appearance.default
```

Comment:
This is named a bit odd -- it's "notEmbdeded", but we are referencing `appearance.embeddedPaymentElement.row.style` ?

Reply:
Renamed!